src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,64 @@
+package mate.jdbc.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.dao.CarDaoImpl;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao = new CarDaoImpl();

```

Comment:
```suggestion
    private CarDao carDao;
```
You don`t need inizialize this field, injector do this